packages/stark-rbac/tslint.json

```diff
@@ -1,5 +1,11 @@
 {
-  "extends": ["tslint:latest", "tslint-sonarts", "codelyzer", "tslint-config-prettier", "../stark-build/config/tslint.json"],
+  "extends": [
+    "tslint:latest",
+    "tslint-sonarts",
+    "tslint-config-prettier",
```

Comment:
This `tslint-config-prettier` should be at the end because it will override any conflicting rule that is managed by Prettier ;)